suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraph.tsx

```diff
@@ -55,7 +56,7 @@ export const PortfolioGraph = forwardRef<PortfolioGraphRef>((_props, ref) => {
 
     return (
         <VStack spacing="large" testID="@home/portfolio/graph">
-            {isAnyMainnetAccountPresent ? <PortfolioGraphHeader /> : null}
+            {isAnyMainnetAccountPresent ? <PortfolioGraphHeader /> : null}{' '}
```

Comment:
`{' '}` was intentional?
```suggestion
            {isAnyMainnetAccountPresent ? <PortfolioGraphHeader /> : null}
```

Reply:
Idk, maybe @yanascz knows?

Reply:
The empty text value should not be necessary, if I remember correctly.

Reply:
@Nodonisko It is added in this PR, not by @yanascz. And it also triggers `Error: Text strings must be rendered within a <Text> component.` So I'm pretty sure it was not intentional...

Reply:
Sorry I was confused I thought it was there and I removed it.